dev/travis/script.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Copyright © 2015 Magento. All rights reserved.
+# See COPYING.txt for license details.
+
+set -e
+export PATH="./../../../vendor/bin:$PATH"
+
+case $TEST_SUITE in
+    unit)
+        cd dev/tests/unit
+        phpunit
+        ;;
+    integration)
+        cd dev/tests/integration
+        phpunit
+        ;;
+    static)
+        cd dev/tests/static
+        phpunit --filter 'Magento\\Test\\Php\\LiveCodeTest::(testCodeStyle|testAnnotationStandard)'
```

Comment:
These tests were intentionally broken down into two parts because they took much time. Right now they are fast because, looks like, they just don't test anything. So I'd leave these tests separate, as it may happen that they will become slow again when fixed.